agentlocal/agentlocal.go

```diff
@@ -70,6 +72,7 @@ type Status struct {
 	Connected        bool          `json:"connected"`
 	ServerClockDrift time.Duration `json:"server_clock_drift,omitempty"`
 	ServerLatency    time.Duration `json:"server_latency,omitempty"`
+	PMMVersion       string        `json:"pmm_version,omitempty"`
```

Comment:
I would suggest to not add this field to this struct. It can be added as separate field in status.go

Reply:
Moved into status.go.